src/modules/CSADigitizer/README.md

```diff
@@ -32,16 +33,22 @@ If this behavior is not desired, the `ignore_polarity` parameter can be set to c
 * `clock_bin_tot` : Duration of a clock cycle for the time-over-threshold (ToT) clock. If set, the output charge is delivered as time over threshold in units of ToT clock cycles, otherwise the pulse integral is stored instead.
 
 #### Parameters for the simplified model
+* `feedback_capacitance` : The feedback capacity to the amplifier circuit. Defaults to 5e-15 F.
 * `rise_time_constant` : Rise time constant of CSA output. Defaults to 1 ns.
 * `feedback_time_constant` : Feedback time constant of CSA output. Defaults to 10 ns.
 
 #### Parameters for the CSA model
+* `feedback_capacitance` : The feedback capacity to the amplifier circuit. Defaults to 5e-15 F.
 * `krummenacher_current` : The feedback current setting of the CSA. Defaults to 20 nA.
 * `detector_capacitance` : The detector capacitance. Defaults to 100 e-15 F.
 * `amp_output_capacitance` : The capacitance at the amplifier output. Defaults to 20 e-15 F.
 * `transconductance` : The transconductance of the CSA feedback circuit. Defaults to 50e-6 C/s/V.
 * `temperature` : Defaults to 293.15K.
 
+#### Parameters for the custom model
+* `response_function` : A 1-dimensional [`ROOT::TFormula`](https://root.cern.ch/doc/master/classTFormula.html) expression for the impulse response function.
+* `response_parameters` : Array of the parameters in the repsonse function.
```

Comment:
Typo `repsonse`.

Maybe add that the number of parameters needs to match the one given in the formula?